tests/transform/index.js

```diff
@@ -185,10 +135,11 @@ test('babel-compiled template literals', function (t) {
       path.join(__dirname, '../../')
     ]
   })
+  b.require('./lib/createElement.js', { expose: 'nanohtml/lib/createElement' })
   b.bundle(function (err, src) {
     fs.unlinkSync(FIXTURE)
     t.ifError(err)
-    t.ok(src.indexOf('document.createElement("div")') !== -1, 'created a tag')
+    t.ok(src.indexOf('html.createElement("div",{"class":"whatever "+abc},[xyz])') !== -1, 'created a tag')
```

Comment:
to only test if the tag is being created checking `html.createElement("div",` would be good but, not a big deal

Reply:
Ok